2nd.py

```
@@ -0,0 +1,11 @@
+# simple_utils.py - A tiny utility library
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)